src/palette/qml/MuseScore/Palette/PalettePropertiesDialog.qml

```diff
@@ -156,13 +181,18 @@ StyledDialogView {
             onClicked: {
                 propertiesModel.showGrid = !checked
             }
+
+            navigation.panel: root.navigationPanel
+            navigation.order: 2 + repeater.count
         }
 
         ButtonBox {
             width: parent.width
 
             buttons: [ ButtonBoxModel.Cancel, ButtonBoxModel.Ok ]
 
+            navigationPanel.section: root.navigationSection
```

Comment:
Shouldn't we set the order too, so that it doesn't collide with the `navigationPanel` of the other items?